<?xml version="1.0"?>
<!--
Launch the path follower with the kinematic controller for a unicycle robot.
-->
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="unicycle_inputscaling">
                <param name="vehicle_length" value="0.5" />
                <param name="k" value="2.0" />
                <param name="max_angular_velocity" value="0.8" />
            </group>
        </group>
    </group>
</launch>
